GVFS/GVFS.FunctionalTests/Tests/GitCommands/GitCommandsTests.cs

```diff
@@ -557,6 +557,18 @@ public void MoveFolderCommitChangesSwitchBranchSwitchBackTest()
         {
             this.CommitChangesSwitchBranchSwitchBack(fileSystemAction: this.MoveFolder);
         }
+
+        // MacOnly because Windows does not support file mode
+        [TestCase]
+        [Category(Categories.MacOnly)]
+        public void UpdateFileModeOnly()
+        {
+            const string TestFileName = "test-file-mode";
+            this.CreateFile("#!/bin/bash\n", TestFileName);
+            this.ChangeMode(TestFileName, Convert.ToInt32("755", 8));
+            this.ValidateGitCommand($"add {TestFileName}");
+            this.ValidateGitCommand($"ls-files --stage {TestFileName}");

```

Comment:
Did you confirm this test fails without your change?

Reply:
Yep.